http/task_service_test.go

```diff
@@ -38,6 +39,20 @@ func NewMockTaskBackend(t *testing.T) *TaskBackend {
 			FindOrganizationF: func(ctx context.Context, filter platform.OrganizationFilter) (*platform.Organization, error) {
 				org := &platform.Organization{}
 				if filter.Name != nil {
+					if *filter.Name == "non-existant-org" {
+						return nil, &platform.Error{
+							Err:  errors.New("org not found or unauthorized"),
+							Msg:  "org " + *filter.Name + " not found or unauthorized",
+							Code: platform.ENotFound,
+						}
+					}
+					if *filter.Name == "unauthorized" {
```

Comment:
It doesn't look like the `*filter.Name == "unauthorized"` path is exercised. It should be exercised or removed.